cpp/src/text/subword/load_merges_file.cu

```diff
@@ -119,7 +119,7 @@ std::unique_ptr<detail::merge_pairs_map_type> initialize_merge_pairs_map(
 
   merge_pairs_map->insert(iter,
                           iter + input.size(),
-                          cuco::murmurhash3_32<cudf::hash_value_type>{},
+                          thrust::identity<cudf::hash_value_type>{},
```

Comment:
That's correct. Previously, this was hashing a hashed value. 
Also, so that the same hasher is used for both `insert` and `find` -- a bug that this fixes too.